pkg/compiler/codegen.go

```diff
@@ -230,6 +230,9 @@ func (c *codegen) emitLoadVar(pkg string, name string) {
 	if vi.tv.Value != nil {
 		c.emitLoadConst(vi.tv)
 		return
+	} else if vi.index == -1 {
```

Comment:
That's like a magic index, it deserves a constant probably.

Reply:
Fixed.

Reply:
Was it updated after the fix? I don't see anything changed here.